.golangci.yml

```diff
@@ -41,8 +43,10 @@ linters:
   - nilerr
   - noctx
   - nolintlint
+  - nosprintfhostport
   - prealloc
   - predeclared
+  - reassign
```

Comment:
This needs a pattern in settings to be enabled : https://golangci-lint.run/usage/linters/#reassign

There's a couple of findings when enabled.

Reply:
Removed for now, will try to explore more on pattern and raise as part of a separate PR.